app/controllers/Redirect.scala

```diff
@@ -8,4 +10,8 @@ trait Redirect {
     val queryParamsToForward = Set("INTCMP", "CMP", "REFPVID") ++ additionalParams
     Redirect(destinationUrl, request.queryString.filterKeys(queryParamsToForward), SEE_OTHER)
   }
+
+  def thankYouMobileUri(amount: ContributionAmount): String = {
+    s"x-gu://contribution?date${LocalDate.now().toString}&amount=$amount"
```

Comment:
it's stored and sent back to olgil, so we can decide what to do. For instance if someone pays £1,000 we can decide they get no creative for 10 years

Realisticly we won't use it in the near future, just the date